### PR TITLE
Jebuss/pedestal studies

### PR DESCRIPTION
**Pull request by jebuss**
new Feature: TimeOverThresholdTL allows to calculate the time over Threshold for a given window on the Timeline
FWHMPulses uses Double instead of Int

### CODE REVIEW

**Comment by jebuss** [comment]
@fact-project/fact-tools-core-dev please have a look

**Comment by MaxNoe** [comment]
Please remove getters

File: src/main/java/fact/features/singlePulse/FWHMPulses.java [hunk 1]
```diff
@@ -71,14 +71,14 @@ public Data process(Data input) {
                 double rightAmplitude = maxAmplitude;
                 double leftAmplitude  = maxAmplitude;
 
-                ///ToDo this method is still super fishy, rework it!
+                ///TODO: this method is still super fishy, rework it!
```
Comment (MaxNoe) [inline src/main/java/fact/features/singlePulse/FWHMPulses.java]: This should not be merged then, right?

**Comment by jebuss** [comment]
@fact-project/fact-tools-core-dev adapted all comments, any objections to merge?

**Pull request merged**